pages/web_cloud/domains/dns_server_edit/guide.it-it.md

```diff
@@ -10,94 +10,205 @@ updated: 2024-06-17
+> Ogni riquadro di input (visualizzato nella schermata precedente) può contenere un solo** server DNS alla volta. Un server DNS corrisponde quindi a un riquadro.
 >
+> Inoltre, una nota informativa su sfondo blu, situata sopra il primo riquadro, indica l'intervallo di server DNS che è possibile dichiarare per il tuo nome di dominio. Questi valori variano in base all'estensione del dominio.
+
+Una volta inserite le informazioni, clicca sul pulsante `+`{.action} a destra dei 2 moduli. che permette di aggiungere il server DNS e visualizzare un nuovo riquadro di inserimento sotto quello precedente.
+
+Ripeti l’operazione per tutti i server DNS da aggiungere, rispettando i limiti indicati nella nota informativa.
+Clicca sul pulsante `+`{.action} per ogni server DNS per confermare l’inserimento e l’aggiunta.
+
+Una volta aggiunti tutti i server DNS, clicca su `Applicare la configurazione`{.action}. Viene visualizzata la seguente finestra:
+
+![Modify DNS servers](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-servers/modify-my-dns-servers-2-apply-configuration.png){.thumbnail}
 
-Una volta inserite queste informazioni, clicca su `Applica la configurazione`{.action}. Lo stato dei server DNS verrà aggiornato nella tabella con le nuove informazioni inserite.
+che riepiloga i nomi dei server DNS applicati al dominio.
+Se corrispondono a quelli che vuoi applicare, clicca su `Applica`{.action}.
 
-![dns-server](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-servers/edit-dns-servers.png){.thumbnail}
+I server DNS dichiarati e la configurazione DNS applicata verranno disattivati per il dominio. La zona DNS dichiarata sui tuoi server DNS diventerà la zona DNS attiva per il tuo dominio.
+
+#### Opzione 3 - Utilizza i DNS OVHcloud e i tuoi DNS
+
+![Modify DNS servers](/pages/assets/screens/control_panel/product-selection/web-cloud/domain-dns/dns-servers/modify-my-dns-servers-3.png){.thumbnail}
+
+Questa opzione permette di combinare l'utilizzo dei propri server DNS conservando i server DNS OVHcloud attivi per il dominio. Questa combinazione permette, ad esempio, di assicurare un maggiore accesso ai diversi servizi associati al dominio (hosting Web, server di posta, ecc...). Infatti, se un gruppo di server DNS diventa indisponibile per qualche minuto, gli altri server DNS dichiarati possono subentrare.
+
+e verifica che le configurazioni delle zone DNS dei diversi server siano configurate correttamente per funzionare tutte insieme. Nella maggior parte dei casi, tutti i server DNS saranno operativi. Saranno tutti in grado di rispondere alle richieste che verranno loro fatte in modo casuale sulla rete DNS.
+
+> [!warning]
+>
+> 1. Presta particolare attenzione se decidi di utilizzare quest’ultima opzione. richiede infatti conoscenze avanzate sul funzionamento della rete DNS, dei server DNS e delle zone DNS.
+>
```

Comment:
To make sure your numbered list is not broken on prod (1,1,1 instead of 1,2,3), delete empty lines and add `<br>`